.github/config/linux-evergreen.json

```diff
@@ -8,10 +8,18 @@
   "num_gtest_shards": 1,
   "targets": [
     "base",
+    "base_perftests",
     "base_unittests",
+    "boringssl_ssl_tests",
+    "crypto_unittests",
+    "ipc_tests",
     "net",
     "net_unittests",
-    "nplb"
+    "nplb",
+    "sql_unittests",
+    "trace_processor_minimal_smoke_tests",
+    "url_unittests",
+    "zlib_unittests"
```

Comment:
These targets should be added to the test_target.json file for linux-evergreen.

Reply:
Does it matter when we're not yet running them? I like having a singular list right now to make it clear what can and can't build

Reply:
Oh, I thought the intention was to run them. If you only want to build them then this is the correct list.